File: programming-language-1/c++/lab/work/Atividade Lab 3/ContaEspecial/classes/Conta/Conta.cpp
#include "Conta.h"

// Conta
Conta::Conta() {
    saldo = 0.00;
}

void Conta::setSaldo(double n_saldo) {
    saldo = n_saldo;
}

double Conta::getSaldo() {
    return saldo;
}

void Conta::deposita(double valorDeposito) {
    setSaldo(getSaldo() + valorDeposito);
}

bool Conta::saca(double valorSacar) {
    double n_saldo;

    if(getSaldo() >= valorSacar) {
        n_saldo = getSaldo() - valorSacar;
        setSaldo(n_saldo);
        return true;
    } else {
        return false;
    }
}

File: programming-language-2/_old/lab/aula5/SistemaBancarioComDao/src/banco/dominio/Conta.java
package banco.dominio;

import java.math.BigDecimal;

public class Conta {

    private long numero;
    private BigDecimal saldo;

    public Conta(long numero, BigDecimal saldo) {
        this.numero = numero;
        this.saldo = saldo;
    }

    public long getNumero() {
        return numero;
    }

    public void setNumero(long numero) {
        this.numero = numero;
    }

    public BigDecimal getSaldo() {
        return saldo;
    }

    public void setSaldo(BigDecimal saldo) {
        this.saldo = saldo;
    }
}
